### ISSUE TITLE
StackOverflow error in printf

### ISSUE THREAD

**Issue by ikirill** [JeffreySarnoff/ArbNumerics.jl#14]
Using master:
```
  [7e558dbc] ArbNumerics v0.3.8 #master (https://github.com/JeffreySarnoff/ArbNumerics.jl.git)
```

```
julia> using ArbNumerics

julia> @sprintf "%.7f" ArbFloat(pi)
ERROR: StackOverflowError:
Stacktrace:
 [1] fix_dec(::ArbFloat{128}, ::Int64) at ./printf.jl:989 (repeats 80000 times)
```

**Comment by JeffreySarnoff** [comment]
What version of Julia are you using?

**Comment by JeffreySarnoff** [comment]
While I look into it, I suggest
```julia
using Printf
@sprintf "%.7f" BigFloat(ArbFloat(pi))
```

**Comment by JeffreySarnoff** [comment]
fixed in v0.3.9 see https://github.com/JeffreySarnoff/ArbNumerics.jl/blob/master/docs/src/seeingvalues.md
